.env.test

```diff
@@ -20,4 +20,7 @@ API_SITE_ID=nmdc-runtime
 API_SITE_CLIENT_ID=chws-tk74-51
 API_SITE_CLIENT_SECRET=070d56033559e3096d4f82d30aabdd51c3b8a57ac20d24f743a27444d5edc3db
 
-MINTING_SERVICE_ID=nmdc:minter_service_11
\ No newline at end of file
+MINTING_SERVICE_ID=nmdc:minter_service_11
+
+# Can append e.g. '/nmdc/biosample_set.bson.gz' to fetch the 'biosample_set' collection from the 'nmdc' database.
+MONGO_REMOTE_DUMP_URL_PREFIX=https://nmdc-runtime.files.polyneme.xyz/nmdc-mongodumps/dump_nmdc-prod_2025-02-12_20-12-02
```

Comment:
@shreddd @eecavanna here's the environment variable I referred to during the recent NMDC infrastructure sync, to point to a mirror of a subset of e.g. the `dump_nmdc-prod_2025-02-12_20-12-02` directory within NMDC's project directory on the NERSC filesystem.